test/server/models/UserModel.js

```diff
@@ -0,0 +1,115 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+
+import db from '../../../server/app/db/models/index';
+import { userData } from '../TestData';
+
```

Comment:
Nice Idea with the spacing, but, do you really need to leave 2 blank lines ? You can also ensure consistent spacing between your test suites and test cases.